workflow/workflow_commands.go

```diff
@@ -595,7 +595,7 @@ func queryWorkflowHelper(c *cli.Context, queryType string) error {
 	if queryResponse.QueryRejected != nil {
 		fmt.Printf("Query was rejected, workflow has status: %v\n", queryResponse.QueryRejected.GetStatus())
 	} else {
-		queryResult := stringify.AnyToString(queryResponse.QueryResult, true, 0, dataconverter.CustomDataConverter())
+		queryResult := stringify.AnyToString(queryResponse.QueryResult, true, 0, converter.GetDefaultDataConverter())
```

Comment:
Correctness: `queryResponse` was obtained using the frontend client, which was created using the usual factory, which sets the gRPC interceptor: https://github.com/temporalio/cli/blob/80a17b4de02405e0edc646754b7d48094bf51095/workflow/workflow_commands.go#L550